### PR TITLE
Add innosetup script file for creating installer

### PR DESCRIPTION
**Pull request by danparisd**
This worked on the dev branch and is based off that. Assuming the base still builds it should work fine.

### CODE REVIEW

File: installerscript.iss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "USFM Quick Converter"
+#define MyAppVersion "0.0.1"
+#define MyAppPublisher "Wycliffe Associates"
+#define MyAppURL "https://wycliffeassociates.org"
+#define MyAppExeName "Hackathon_Converter.exe"
```
Comment (PurpleGuitar) [inline installerscript.iss]: I recommend we rename the executable to USFM_Converter, so it's not confusing to our users.

Comment (danparisd) [reply installerscript.iss]: done. that would actually be needed for compatibility as well. Good catch, thanks.

**Comment by PurpleGuitar** [comment]
I like the change to the packager, but I think you'll also need to change the executable name in your solution file, e.g. https://github.com/WycliffeAssociates/quick_usfm_converter/blob/3117067a82eceff1227c273074b3baab75821ca0/Hackathon_Converter/Hackathon_Converter.csproj#L10

**Comment by danparisd** [comment]
> 
> 
> I like the change to the packager, but I think you'll also need to change the executable name in your solution file, e.g.
> 
> https://github.com/WycliffeAssociates/quick_usfm_converter/blob/3117067a82eceff1227c273074b3baab75821ca0/Hackathon_Converter/Hackathon_Converter.csproj#L10

Indeed. Brian already changed it in the solution that's been merged to development.

File: installerscript.iss (hunk 1 above)
Comment (danparisd) [reply installerscript.iss]: @PurpleGuitar Can you resolve this change request so @BryanHo10 can merge these changes and we can get the installer built from the dev branch?

Comment (PurpleGuitar) [reply installerscript.iss]: Yes, approved!  Sorry for the delay.

**Pull request merged**